client/state/ui/editor/media/actions.js

```diff
@@ -1,11 +1,56 @@
 /**
  * Internal dependencies
  */
-import { EDITOR_MEDIA_EDIT_ITEM_SET } from 'state/action-types';
+import {
+	EDITOR_MEDIA_EDIT_ITEM_SET,
+	IMAGE_EDITOR_ROTATE,
+	IMAGE_EDITOR_SCALE,
+	IMAGE_EDITOR_SET_FILE_INFO,
+	IMAGE_EDITOR_STATE_RESET
+} from 'state/action-types';
 
 export function setEditorMediaEditItem( item ) {
 	return {
 		type: EDITOR_MEDIA_EDIT_ITEM_SET,
 		item
 	};
 }
+
+export function resetImageEditorState() {
+	return {
+		type: IMAGE_EDITOR_STATE_RESET
+	};
+}
+
+export function imageEditorRotate( degrees ) {
+	return ( dispatch, getState ) => {
+		const state = getState(),
+			imageEditorState = state.ui.editor.media.imageEditor;
```

Comment:
Let's avoid accessing state from our actions. We can instead dispatch our intent to rotate our image -90 degrees relative to the current position. This might be as simple as:
```
return {
   type: IMAGE_EDITOR_ROTATE_COUNTERCLOCKWISE
};
```